### PR TITLE
Adding support for multisig wallet

### PR DESCRIPTION
**Pull request by Danswar**
Current progress:
- Adding multi signature wallet from Home Screen.
- Sharing and managing keys.
- Check balances and transactions.
- Create transaction.

Missing:
- [x]  Load transaction from "Scan" button
- [x] Use main Bitcoin wallet as seed for Key 1 in Multisig Vault
- [x] Key 2 and 3 must be XPub only, seed phrases should be rejected when scanned or input it manually.
- [x] Remove the "Manage Key" button as it no longer serve any purpose.

### CODE REVIEW

**Comment by TaprootFreak** [comment]
Looks good ✅

More things are needed:
When creating the vault, the user should not be asked for Vault Key 1. 
Vault key 1 is always the seed of the device. 
The DFX Taro wallet already has a seed for the Bitcoin on-chain wallet. This seed should always automatically be Key 1 for the MultiSig, without the user being able to select it.

**Comment by TaprootFreak** [comment]
Another requirement:

No seeds may be stored for MultiSig Key 2 and 3. XPub only. 
If someone enters 12 words, it should be rejected.
If someone scans a seed QR code with the scan function, it should be rejected. 
Key 1 is always the seed from the existing Bitcoin OnChain wallet. And Key 2 and 3 must always be an XPub. Mandatory.

**Comment by TaprootFreak** [comment]
Another requirement:

If key 1 is always the seed of the Bitcoin OnChain wallet, and keys 2 and 3 are always just an Xpub, then we can remove this "Manage Key" button. This button then no longer serves any purpose. 

photo_2024-04-30 14 40 54

**Comment by TaprootFreak** [comment]
I have just tested the APK and it works correctly. 

Further improvements will follow, but this PR can be merged from my side. 

Currently only the review from @davidleomay  is still pending.

**Comment by Danswar** [comment]
closes #40

**Comment by TaprootFreak** [comment]
Just tested and it works as expected. 

further improvements will follow in a new PR

File: screen/send/details.js [hunk 1]
```diff
@@ -174,7 +174,9 @@ const SendDetails = () => {
       navigation.goBack();
       return;
     }
-    const newWallet = (routeParams.walletID && wallets.find(w => w.getID() === routeParams.walletID)) || suitable[0];
+    const walletFromParmas = routeParams.walletID && wallets.find(w => w.getID() === routeParams.walletID);
+    const isWalletFromParamsSuitable = walletFromParmas && walletFromParmas.chain === Chain.ONCHAIN && walletFromParmas.allowSend();
```
Comment (davidleomay) [inline screen/send/details.js:178]: Typo 😉

**Review by davidleomay — APPROVED** [review]

**Pull request merged**